src/utils/backend.js

```diff
@@ -111,7 +111,13 @@ export default class Backend {
 
   static getTokenInfo = async () => backendFetch('tokenCache/tokenInfo');
 
-  static getWordRegistry = async (ordering, direction, search) => backendFetch(`tokenCache/wordRegistry?ordering=${ordering}&direction=${direction}&search=${search || ''}`);
+  static getWordRegistry = async (ordering, direction, search) => {
+    const queryParams = new URLSearchParams();
+    if(ordering) queryParams.set('ordering', ordering);
+    if(direction) queryParams.set('direction', direction);
+    if(search) queryParams.set('search', search);
```

Comment:
```suggestion
    if (ordering) queryParams.set('ordering', ordering);
    if (direction) queryParams.set('direction', direction);
    if (search) queryParams.set('search', search);
```
this should fix linter errors: https://github.com/aeternity/superhero-ui/pull/1029/checks?check_run_id=2038213738#step:7:1098